<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoProdutos</class>
 <widget class="QDialog" name="InfoProdutos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cupcake Store - Produtos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursoss.qrc">
    <normaloff>:/Imagens/imgs/ee.png</normaloff>:/Imagens/imgs/ee.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: century gothic;
	font-size: 16px;
	background: white;
}

QToolButton
{
	background: transparent;
	border-radius: 40px;
}

QPushButton
{
	background: orange;
	border-radius: 40px;
}

Qlabel
{
	color: white;
	background: transparent;
}

QPushButton
{
	color: black;
}

QPushButton:hover
{
	color: #333;
	border-radius:15px;
	background: #49ebff;
}

QLineEdit
{
	background: orange;
	border: none;
	color: black;
	border-bottom: 1px solid white;
}

</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_novoProduto">
    <attribute name="title">
     <string>Novo Produto</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>707</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Código produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_codigoProduto"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Descrição produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_descricaoProduto"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Quantidade em estoque</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_qtdeEstoque"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Valor de compra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_valorCompra"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Valor de venda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_valorVenda"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_novoProduto">
          <property name="text">
           <string>Limpar campos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_gravarProduto">
          <property name="text">
           <string>Gravar produto</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="recursoss.qrc">
       <normaloff>:/Imagens/imgs/ee.png</normaloff>:/Imagens/imgs/ee.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>360</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="recursoss.qrc">
       <normaloff>:/Imagens/imgs/ee.png</normaloff>:/Imagens/imgs/ee.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>360</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="recursoss.qrc">
       <normaloff>:/Imagens/imgs/ee.png</normaloff>:/Imagens/imgs/ee.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_gestaoProdutos">
    <attribute name="title">
     <string>Gestão Produtos</string>
    </attribute>
    <widget class="QTableWidget" name="tab_GPprodutos">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>291</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Ordenar por:</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QRadioButton" name="radbtn_GPcodigo">
         <property name="text">
          <string>Código</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radbtn_GPnome">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>234</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Código do produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_GPcodigo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_GPproduto"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Quantidade</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_GPqtde"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Valor de compra</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_GPvalorCompra"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Valor de venda</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_GPvalorVenda"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_GPalterar">
          <property name="text">
           <string>Alterar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_GPexcluir">
          <property name="text">
           <string>Excluir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>360</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="recursoss.qrc">
       <normaloff>:/Imagens/imgs/ee.png</normaloff>:/Imagens/imgs/ee.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>441</x>
       <y>31</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Pesquisar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_GPfiltrar"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_GPfiltrar">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Filtrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>400</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Voltar</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>txt_codigoProduto</tabstop>
  <tabstop>txt_descricaoProduto</tabstop>
  <tabstop>txt_qtdeEstoque</tabstop>
  <tabstop>txt_valorCompra</tabstop>
  <tabstop>txt_valorVenda</tabstop>
  <tabstop>btn_gravarProduto</tabstop>
  <tabstop>btn_novoProduto</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>toolButton_4</tabstop>
  <tabstop>toolButton_2</tabstop>
  <tabstop>tab_GPprodutos</tabstop>
  <tabstop>radbtn_GPcodigo</tabstop>
  <tabstop>radbtn_GPnome</tabstop>
  <tabstop>txt_GPcodigo</tabstop>
  <tabstop>txt_GPproduto</tabstop>
  <tabstop>txt_GPqtde</tabstop>
  <tabstop>txt_GPvalorCompra</tabstop>
  <tabstop>txt_GPvalorVenda</tabstop>
  <tabstop>btn_GPalterar</tabstop>
  <tabstop>btn_GPexcluir</tabstop>
  <tabstop>toolButton_3</tabstop>
  <tabstop>txt_GPfiltrar</tabstop>
  <tabstop>btn_GPfiltrar</tabstop>
 </tabstops>
 <resources>
  <include location="recursoss.qrc"/>
 </resources>
 <connections/>
</ui>
